src/sdk/map_api_types.js

```diff
@@ -13,7 +13,7 @@ const { new_object_id, parse_object_id } = require('../util/mongo_utils');
 const undefined_id = undefined;
 
 /**
- * 
+ *
```

Comment:
Maybe you have a setting for your editor/formatter is to skip formatting whitespace in comments?
FWIW here is my .jsbeautifyrc:
```
{
  "indent_size": 4,
  "indent_char": " ",
  "indent_level": 0,
  "indent_with_tabs": false,
  "brace_style": "collapse,preserve-inline",
  "preserve_newlines": true,
  "max_preserve_newlines": 10,
  "end_with_newline": true
}
```

Reply:
My setting have trailing whitespaces removed. If you don't want that, I can rollback the comment lines having trailing whitepaces removed.

Reply:
I don't mind just consider that the project does not have a standard around whitespace in comments so it might make your PR's contain more whitespace changes. Resolving.